client/src/main/java/org/apache/nemo/client/JobLauncher.java

```diff
@@ -381,6 +387,39 @@ private static Configuration getDriverConf(final Configuration jobConf) throws I
       .set(DriverConfiguration.DRIVER_IDENTIFIER, jobId)
       .set(DriverConfiguration.DRIVER_MEMORY, driverMemory)
       .build();
+
+    final JavaConfigurationBuilder jcb = Tang.Factory.getTang().newConfigurationBuilder();
+    jcb.bindSetEntry(DriverIdleSources.class, LambdaIdlenessSource.class);
```

Comment:
LambdaIdleSource should be configured when users want to user lambda. Please fix this code